test/ExpressionSet_basic_test.py

```diff
@@ -1,37 +1,69 @@
 """Basic ExpressionSet tests."""
+from test.common_test import check_get_set, check_save_set_output
 from test.util import log_this
+from typing import Any
 
 import pytest
-from SetAPI.generic.constants import INC_ITEM_INFO, INC_ITEM_REF_PATHS
+from SetAPI.expression.ExpressionSetInterfaceV1 import ExpressionSetInterfaceV1
+from SetAPI.generic.constants import INC_ITEM_INFO, INC_ITEM_REF_PATHS, REF_PATH_TO_SET
 from SetAPI.SetAPIImpl import SetAPI
-from SetAPI.util import info_to_ref
 
 DEBUG = False
 
+API_CLASS = ExpressionSetInterfaceV1
+SET_TYPE = "expression"
 
-def test_save_expression_set(
+
+def save_expression_set(
     set_api_client: SetAPI,
     context: dict[str, str | list],
     ws_id: int,
-    expression_refs: list[str],
-) -> None:
-    expression_set_name = "test_expression_set"
-    expression_items = [{"label": "foo", "ref": ref} for ref in expression_refs]
-    expression_set = {"description": "test_expressions", "items": expression_items}
-    result = set_api_client.save_expression_set_v1(
+    set_name: str,
+    set_data: dict[str, Any],
+) -> dict[str, Any]:
+    """Given a set_name and set_data, save an expression set."""
+    return set_api_client.save_expression_set_v1(
         context,
         {
             "workspace_id": ws_id,
-            "output_object_name": expression_set_name,
-            "data": expression_set,
+            "output_object_name": set_name,
+            "data": set_data,
         },
     )[0]
-    assert result is not None
-    assert "set_ref" in result
-    assert "set_info" in result
-    assert result["set_ref"] == info_to_ref(result["set_info"])
-    assert result["set_info"][1] == expression_set_name
-    assert "KBaseSets.ExpressionSet" in result["set_info"][2]
+
+
+@pytest.fixture(scope="module")
+def expression_set(
```

Comment:
Wow, this structure is starting to look familiar...

Yes, it's going to be refactored away in the near future. Hurrah!